requirements.txt

```diff
@@ -1,17 +1,19 @@
-boto3>=1.1.1
-csvkit>=0.9.1
+boto3==1.1.1
+csvkit==0.9.1
 invoke==0.10.1
 pandas
 # intentionally leaving pandas version blank, b/c `pip install` generally
 # fails on Linux - must use the package manager version
-psycopg2>=2.6.1
-requests>=2.7.0
-gunicorn>=19.3.0
+psycopg2==2.6.1
+requests==2.7.0
+gunicorn==19.3.0
 cryptography>=1.0
-Flask-Cors>=2.1.0
-Flask-BasicAuth>=0.2.0
-marshmallow>=2.0.0.b4
-marshmallow-sqlalchemy>=0.4.0
+Flask==0.10.1
+Flask-SQLAlchemy==1.0
```

Comment:
I added this dependency from [sandman2's `requirements.txt`](https://github.com/18F/sandman2/blob/table-delete/requirements.txt) to pin it, as multiple deprecation warnings were being logged when using the latest version of Flask-SQLAlchemy.